lightning/src/chain/keysinterface.rs

```diff
@@ -245,12 +247,55 @@ pub trait ChannelKeys : Send+Clone {
 	/// return value must contain a signature.
 	fn sign_local_commitment_htlc_transactions<T: secp256k1::Signing + secp256k1::Verification>(&self, local_commitment_tx: &LocalCommitmentTransaction, local_csv: u16, secp_ctx: &Secp256k1<T>) -> Result<Vec<Option<Signature>>, ()>;
 
+	/// Create a signature for a justice transaction spending a revokable output on a revoked
```

Comment:
This is still super heavy on lightning jargon - maybe say "Create a signature for a transaction spending an HTLC or commitment transaction output when our counterparty broadcast an old state."